Sources/tart/Commands/Run.swift

```diff
@@ -36,6 +36,16 @@ struct Run: AsyncParsableCommand {
   @Flag(help: "Force open a UI window, even when VNC is enabled.")
   var graphics: Bool = false
 
+  @Flag(help: ArgumentHelp(
+    "Pass audio through to host.",
+    discussion: "This is the default if opening a UI window."))
+  var audio: Bool = false
+
+  @Flag(help: ArgumentHelp(
+    "Don't pass audio through to host.",
+    discussion: "Disables audio pass through, even when opening a UI window."))
+  var noAudio: Bool = false
```

Comment:
There is no need in `noAudio`. SwiftArguments will handle `--no-audio` if you have `--audio`. You can leave only `audio` variable and default it to `true` for backward compatibility.

Reply:
Ah, great! I modeled the two variables after `graphics` and `noGraphics`.

Reply:
Oh. Maybe I confused SwiftArguments features with what we use for Go. Regardless, I think we can leave just one variable either `audio = true` or `noAudio = false`. And pass it down to `craftConfiguration`.